setup.py

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env python
-"""A setuptools-based script for installing NailGun."""
+"""A setuptools-based script for installing NailGun.
+
+For more information, see:
+
+* https://packaging.python.org/en/latest/index.html
+* https://docs.python.org/distutils/sourcedist.html
```

Comment:
This link explains the `MANIFEST.in` file. The Python 2.7 and 3.4 docs are significantly different, but they both cover the same material.